docs/development.rst

```diff
@@ -455,18 +455,21 @@ should be done:
 
        npm install -g 'npm@>=5.6'
 
-2. Update ``dependencies`` or ``devDependencies`` in ``girder/web_client/package.json.template``
-   to add a new *abstract* specifier for the package:
+2. Update ``girder/web_client/package.json.template`` or ``girder/web_client/src/package.json`` to
+   add a new *abstract* specifier for the package:
 
-  * Packages that are bundled into the web client should generally use the
+  * Packages that are bundled into the web client must be listed under the ``dependencies`` field
+    of ``girder/web_client/src/package.json`` and should generally use the
```

Comment:
Should this now just be done with the `npm add` command?

Reply:
There is no `npm add`, just an `npm install --save`. We don't want that, because there's no need to create a `node_modules` or `package-lock.json` within the `girder/web_client/src/` directory, which the command will always do. Since we don't need the functionality to keep those things in sync, hand-editing the file is going to be much easier than working with the statefulness of `npm install`.

Reply:
Makes sense, thanks.